src/node/constitution.h

```diff
@@ -4,5 +4,5 @@
 
 namespace ccf
 {
-  using Constitution = kv::JsonSerialisedMap<size_t, std::string>;
+  using Constitution = ServiceMap<size_t, std::string>;
```

Comment:
`read_ledger.py` made it trivial to test this, without a bunch of trial-and-error. I think this was `JsonSerialisedMap` rather than `ServiceMap` purely by accident? And now that its accessed from JS, its much easier if it is `ServiceMap` like everything else.

The KV writes to `public:ccf.gov.constitution` from the genesis transaction (with a string constructed in `cchost`) and from this new proposal (with a string constructed in `proposal_generator.py`) are now byte-identical.

Reply:
Spotted by @letmaik, I've included the same change in https://github.com/microsoft/CCF/pull/2442